cli/src/main/kotlin/OrtMain.kt

```diff
@@ -71,7 +71,10 @@ class OrtMain : CliktCommand(name = ORT_NAME, epilog = "* denotes required optio
 
     private val stacktrace by option(help = "Print out the stacktrace for all exceptions.").flag()
 
-    private val configArguments by option("-P", help = "Allows to override configuration parameters.").pair().multiple()
+    private val configArguments by option("-P", help = "Allows to override configuration parameters.")
```

Comment:
Can you also align the help text with the one used below?

Reply:
Done.

Reply:
I guess @mnonnenmacher meant the wording of the help text, not the formatting of it. Correct @mnonnenmacher? Because so far the option-argument formatting in `OrtMain` was a bit different to the one used in sub-commands in that it were one-liners, and I think it should stay like this throughout the file.

Reply:
How would you like to have it formatted exactly? (I'm asking as the help text does not fit into one line anymore so I don't see how to make it a one-liner.

Reply:
Yeah, scratch that. My comment was only valid for the first commit, but would have been invalidated for the second commit. I've approved.

Reply:
Ah, sorry. I though it was meant for the 2nd one.